### PR TITLE
Revisions to crowdsourcing tutorial

### PR DESCRIPTION
**Pull request by bhancock8**
Made a pass over the Crowdsourcing Tutorial.
- Mostly local edits
- A couple renamings ("answers" -> "labels", "crowd worker" -> "crowdworkers", etc.)
- Simplify to one LF per crowdworker instead of splitting in 2

Some markdown diffs will appear larger than they are where I split on sentence boundaries instead of characters for clearer diffs in the future.

**Test plan:**
`tox -e crowsourcing`

### CODE REVIEW

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 1]
```diff
@@ -295,15 +273,16 @@ def encode_text(text):
 sklearn_model = LogisticRegression(solver="liblinear")
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
```
Comment (brahmaneya) [inline crowdsourcing/crowdsourcing_tutorial.py]: We can use label_model.predict instead of label_model.predict_proba followed by probs_to_preds

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 2]
```diff
@@ -295,15 +273,16 @@ def encode_text(text):
...
 
 # %% [markdown]
 # ## Summary
 #
 # In this tutorial, we accomplished the following:
-# * We showed how Snorkel can handle crowdsourced labels, combining them with other programmatic LFs to improve coverage.
-# * We showed how the LabelModel learns to combine inputs from crowd workers and other LFs by appropriately weighting them to generate high quality probabilistic labels.
-# * We showed that a classifier trained on the combined labels can achieve a fairly high accuracy while also generalizing to new, unseen examples.
+# * We demonstrated how to combine crowdsourced labels with other programmatic LFs to improve coverage.
+# * We used the `LabelModel` to learn how to combine inputs from crowdworkers and other LFs to generate high quality probabilistic labels.
```
Comment (brahmaneya) [inline crowdsourcing/crowdsourcing_tutorial.py]: I don't think we should say 'to learn how'. Just 'We used the `LabelModel` to combine inputs...'

**Comment by bhancock8** [comment]
Comments addressed! Ready for another look.
